modules/tinymce/src/plugins/table/main/ts/ui/Helpers.ts

```diff
@@ -267,17 +267,20 @@ export interface CellData {
   backgroundcolor?: string;
 }
 
-const extractDataFromCellElement = (editor: Editor, elm: HTMLElement, hasAdvancedCellTab: boolean): CellData => {
+const extractDataFromCellElement = (editor: Editor, cell: HTMLTableDataCellElement, column: SugarElement<HTMLTableColElement> | undefined, hasAdvancedCellTab: boolean): CellData => {
```

Comment:
Why do we have an optional argument in the middle of the function instead of the end?

Reply:
Because it seemed more logical to put it beside the cell. I'll move it.